### PR TITLE
player anim + collider surface types

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

File: include/enums.h [hunk 1]
```diff
@@ -4078,16 +4079,23 @@ enum MusicSettingsFlags {
...
     COLLIDER_FLAGS_UPPER_MASK        = 0x7FFFFE00, // map data dumper needs this to be first
-    COLLIDER_FLAGS_WATER_FLOOR       = 0x00000001,
-    COLLIDER_FLAGS_LAVA_FLOOR        = 0x00000002,
-    COLLIDER_FLAGS_SUSHIE_DOCK_WALL  = 0x00000004,
-    COLLIDER_FLAGS_SNOW_FLOOR        = 0x00000008,
-    COLLIDER_FLAGS_10                = 0x00000010,
-    COLLIDER_FLAGS_20                = 0x00000020,
-    COLLIDER_FLAGS_40                = 0x00000040,
-    COLLIDER_FLAGS_80                = 0x00000080,
+    COLLIDER_FLAGS_SURFACE_TYPE      = 0x000000FF,
```
Comment (nanaian) [inline include/enums.h:4098]: Perhaps suffix this name with _MASK to be consistent with the other one

File: src/891b0_len_fb0.c [hunk 2]
```diff
@@ -29,7 +29,7 @@ void func_800EFD08(void) {
         colliderType = D_80109480;
     }
 
-    D_80109480 = get_collider_type_by_id((u16)gCollisionStatus.currentFloor) & 0xFF;
+    D_80109480 = get_collider_flags((u16)gCollisionStatus.currentFloor) & 0xFF;
```
Comment (nanaian) [inline src/891b0_len_fb0.c:32]:
```suggestion
    D_80109480 = get_collider_flags((u16)gCollisionStatus.currentFloor) & COLLIDER_FLAGS_SURFACE_TYPE_MASK;
```

**Review by nanaian — DISMISSED** [review]

**Comment by BowserSlug** [comment]
✅ This PR fixes 2 warnings!

**Review by ethteck — APPROVED** [review]

**Review by ethteck — APPROVED** [review]

File: src/world/action/land.c [hunk 3]
```diff
@@ -112,15 +117,15 @@ void action_update_step_down_land(void) {
     update_locomotion_state();
 }
 
-void func_802B62CC_E24BEC(void) {
+static void action_update_peach_land(void) {
```
Comment (ethteck) [inline src/world/action/land.c:120]: please don't make any functions static - they won't show up in the elf

**Review by ethteck — CHANGES_REQUESTED** [review]

**Review by ethteck — APPROVED** [review]

**Pull request merged**